README.md

```diff
@@ -21,4 +21,33 @@ If you have docker-compose ready to use, you can set up the virtlet dev environm
 ```
 cd contrib/docker-compose
 docker-compose up
+```
+
+## Manual building
+
+You can also build virtlet manually, without using containers. The whole process looks like in any
+other autotools-based project:
```

Comment:
Not exactly. There is no info about how to configure golang (GOPATH, but... it could be automatically set in `Makefile` for project rootdir) and which version of it is supported. Also `configure` does not verify if golang is already installed.

Example how to do this:
https://github.com/coreos/rkt/blob/802bc1f6a5f64a58f39ab9e5305a5e5f28f74960/configure.ac#L11 - function and it's usage:
https://github.com/coreos/rkt/blob/802bc1f6a5f64a58f39ab9e5305a5e5f28f74960/configure.ac#L681